copier/vcs.py

```diff
@@ -10,14 +10,33 @@
 from packaging import version
 from packaging.version import InvalidVersion, Version
 from plumbum import TF, ProcessExecutionError, colors, local
-from plumbum.cmd import git
 
 from .errors import DirtyLocalWarning, ShallowCloneWarning
 from .types import OptBool, OptStr, StrOrPath
 
+
+def get_git():
+    """Gets `git` command, or fails if it's not available"""
+    try:
+        from plumbum.cmd import git
+        return git
```

Comment:
I think this is better, if you don't mind:

```suggestion
        return local["git"]
```